dockstore-common/src/main/java/io/dockstore/common/metrics/Cost.java

```diff
@@ -31,6 +32,7 @@ public class Cost {
     private String currency = DEFAULT_CURRENCY;
 
     @Schema(description = "The numerical value of the cost", example = "5.99")
+    @PositiveOrZero
```

Comment:
Neat, I also noticed that the Javadoc says " null elements are considered valid."
So maybe `@NotNull` as well